spreadsheet/formula/expression.go

```diff
@@ -10,4 +10,6 @@ package formula
 type Expression interface {
 	Eval(ctx Context, ev Evaluator) Result
 	Reference(ctx Context, ev Evaluator) Reference
+	ToString() string
```

Comment:
Maybe String() would be more standard?  That is also part of the Stringer interface (fmt.Stringer)

Reply:
What kind of string is it?  For debugging (format purposes), or some other functional purpose?

Reply:
renamed